lib/client.js

```diff
@@ -1013,13 +1022,15 @@ function _resume (client, sync) {
         return
       }
 
+      const headersTimeout = getRunningRequestOpts(client, kHeadersTimeout)
```

Comment:
```suggestion
    const headersTimeout = request.headersTimeout != null ? request.headersTimeout  : client[kHeadersTimeout]
```

Reply:
The same will be for body timeout, on every line that we need a timeout setting (its about ~4 places)

```
const headersTimeout = request.headersTimeout != null ? request.headersTimeout  : client[kHeadersTimeout]
const bodyTimeout = request.bodyTimeout != null ? request.bodyTimeout  : client[kBodyTimeout]
```

also when someone else will have to get timeout setting in another place in the future, need to remember to add this condition, is this the right thing to do?